enso/experiment/__init__.py

```diff
@@ -228,7 +267,7 @@ def _dump_results(self, results, experiment_name):
         if not experiment_name:
             experiment_name = strftime("%Y-%m-%d_%H-%M-%S", gmtime())
         result_path = os.path.join(RESULTS_DIRECTORY, experiment_name)
-        results = results.dropna()
+#        results = results.dropna()
```

Comment:
I'm not sure why this line existed but it's probably a good idea to remove it.

Reply:
This was causing the timings rows to get dropped because of the empty values for train.